File: 2027-minimum-moves-to-convert-string/2027-minimum-moves-to-convert-string.py
class Solution:
    def minimumMoves(self, s: str) -> int:
        count = 0
        n = len(s)
        i = 0
        while i < n:
            if s[i] == 'X':
                count += 1
                i += 3
            else:
                i += 1
        return count

File: 2027-minimum-moves-to-convert-string/2027-minimum-moves-to-convert-string.java
class Solution {
    public int minimumMoves(String s) {
        int count = 0;
        int n = s.length();
        for (int i = 0; i < n;) {
            if (s.charAt(i) == 'X') {
                count++;
                i += 3;
            } else {
                i++;
            }
        }
        return count;
    }
}